rpc/Serve_test.go

```diff
@@ -0,0 +1,47 @@
+package rpc_test
+
+import (
+	"context"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"capnproto.org/go/capnp/v3/rpc"
+)
+
+//clientHook := {}
+
+func TestServe(t *testing.T) {
+	lis, err := net.Listen("tcp", ":8888")
```

Comment:
We should also `defer lis.Close()` in case something goes wrong in the test, so that we can release the port.